gatherer/src/main/java/de/zalando/pgobserver/gatherer/StatStatementsGatherer.java

```diff
@@ -114,11 +133,12 @@ protected boolean gatherData() {
         }
     }
 
-    public String getQuery() {
+    private String getQuery() {
 //        and not (usename = 'postgres' and upper(query) like 'COPY %') ?
         String sql = "with q_data as (\n" +
                 "      select\n" +
                 "      now() as timestamp,\n" +
+                "      s.userId,\n" +
```

Comment:
Actually this "userId" is really a numeric...so I think wouldn't make to store it like that, better with a real username e.g. one needs to join pg_user